Add NodeSource note for Ubuntu and Debian based systems (#84)
spacebarchat/docs: docs/setup/server/index.md

-   [Python](https://www.python.org/). Version 3.10 or later. Make sure this is executable via `python` in your terminal.  
     (See: `python-is-python3` package)
-   On Linux: `gcc`/`g++`. Packaged with `build-essential` on Debian/Ubuntu and `base-devel` on Arch.

>      (NOTE: Ubuntu or Debian based systems often ship with outdated versions, so we recommend using NodeJS from [NodeSource](https://github.com/nodesource/distributions))

Reviewer: ```suggestion
     (NOTE: Ubuntu/Debian based systems often ship with outdated node versions, you can use [NodeSource](https://github.com/nodesource/distributions) to install updated an updated version.)
```

>      (NOTE: Ubuntu or Debian based systems often ship with outdated versions, so we recommend using NodeJS from [NodeSource](https://github.com/nodesource/distributions))

Reviewer: ```suggestion
     (NOTE: Ubuntu/Debian based systems often ship with an outdated version of node, you can use [NodeSource](https://github.com/nodesource/distributions) to install a newer version.)
```

Reviewer: I think this wording sounds better

Author: I agree

Reviewer: > I agree

could we change `Ubuntu or Debian` to at least `Ubuntu and Debian`?
